src/components/browse/components/FacetList/index.js

```diff
@@ -562,54 +598,67 @@ export class FacetList extends React.PureComponent {
+                                        <i className="icon icon-fw icon-minus fas"/>
+                                    </button>
+                                    : null }
+                                { showClearFiltersButton && typeof onClearFilters === "function" ?
+                                    <button type="button" className="btn btn-outline-light" onClick={onClearFilters} data-tip="Clear all filters">
+                                        <i className="icon icon-fw icon-times fas"/>
+                                    </button>
+                                    : null }
                             </div>
                         </div>
-                        : null }
-                    { staticFacetElements }
+                        {/*
+                        <div className={"col-auto clear-filters-control" + (showClearFiltersButton ? '' : ' placeholder')}>
+                            <a href="#" onClick={onClearFilters} className={"btn clear-filters-btn btn-xs " + clearButtonClassName}>
+                                <i className="icon icon-fw icon-times fas mr-03"/>
+                                <span>Clear All</span>
+                            </a>
+                        </div>
+                        <div className={"col-auto clear-filters-control" + (anyFacetsOpen ? '' : ' placeholder')}>
+                            <a href="#" onClick={onClearFilters} className={"btn clear-filters-btn btn-xs " + clearButtonClassName}>
+                                <i className="icon icon-fw icon-minus fas mr-03"/>
+                                <span>Clear All</span>
+                            </a>
+                        </div>
+                        */}
+                    </div>
+                    <div {...bodyProps}>
+                        { selectableFacetElements }
+                        { staticFacetElements.length > 0 ?
+                            <div className="row facet-list-separator">
+                                <div className="col-12">
+                                    { staticFacetElements.length } Common Properties
+                                </div>
+                            </div>
+                            : null }
+                        { staticFacetElements }
+                    </div>
                 </div>
-            </div>
+                { popoverJSX && popoverTargetRef ? ( /* `rootClose rootCloseEvent="click"` didn't work as props here */
+                    <Overlay show target={popoverTargetRef} flip placement="auto" rootClose rootCloseDisabled={false}>
+                        { popoverJSX }
+                    </Overlay>
+                ) : null }
+            </React.Fragment>
         );
     }
 }
+
+// TODO: Pull out the split terms into own component
+// 2: get ourselves the fieldSchema and pass it down in here.
+
+// function
```

Comment:
I'm not clear on what this means or if it's left over/unnecessary? If it is important, might be good to clean it up a bit.

Reply:
Yeah is some leftover comments. Had noticed that TermsFacet doesn't do a whole lot and thought about combining the files TermFacet, FacetTermsList, and StaticSingleTerm. But on second thought unsure is great idea to make 1 big(ger) file; if anything could create a subfolder 'TermsFacet' and put those other things in it. But not really important enough to do now; maybe if start to add more functionalities to FacetList in near future, idk.